src/status_im/chat/models/loading.cljs

```diff
@@ -92,17 +66,26 @@
                                       :referenced-messages {}
                                       :messages empty-message-map)))
                       {}
-                      (get-all-stored-chats from to))
+                      new-chats)
         chats (merge old-chats chats)]
-    (update-chats-in-app-db cofx chats)))
+    (fx/merge cofx
+              {:db (assoc db :chats chats
+                          :chats/loading? false)}
+              (filters/load-filters)
+              (commands/load-commands commands/register))))
 
+(defn load-chats-from-rpc
+  [cofx from to]
+  (data-store.chats/fetch-chats-rpc cofx {:from 0
+                                          :to 10
+                                          :on-success
+                                          #(re-frame/dispatch
+                                            [:chats-list/load-success %])}))
 (fx/defn initialize-chats
   "Initialize persisted chats on startup"
   [cofx
    {:keys [from to] :or {from 0 to nil}}]
-  (if config/use-status-go-protocol?
-    (load-chats-from-rpc cofx)
-    (initialize-chats-legacy cofx from to)))
+  (load-chats-from-rpc cofx from -1))
```

Comment:
why `-1` ?

Reply:
it's to indicate that we want all the chats, there's pagination but it's not used currently (-1 indicates all the chats)

Reply:
hm can we use nil? i mean just do not pass this param?

Reply:
I can change that, (-1 to indicate no limit is not uncommon in apis though, while 0 might indicate that you want an empty page, `null` makes the parameter `nillable` which might not be desirable with integer data types), but if you don't mind I will do that next time I upgrade `status-protocol-go` (it takes 3 PRs to get stuff in status-react :) , going through 3 review process etc )